blog/views.py

```diff
@@ -109,10 +129,14 @@ def viewpost(request, post_id):
         content = commonmark.commonmark(post.content)
     else:
         content = post.content
+        if post.image != None:
+
+            image = post.image.__str__()
```

Comment:
does this get the base64 url of image? we should be hosting images in the /media path so that people can link to them.

Reply:
This returns a string 'post_images/{$imagefile_name}'. Do I need to use base64 to encode the file name or something?

Reply:
okay, I missunderstood. this currently correct.